core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -190,14 +190,15 @@ class EntityApiServiceSpec extends ApiServiceSpec {
           responseAs[Entity]
         }
 
+        // TODO: does not test that the path we return is correct.  Update this test in the future if we care about that
```

Comment:
No one has complained about this in years, so we don't care enough for a proper test yet.